### PR TITLE
Ecommerce apps missing products bug

### PR DESCRIPTION
**Pull request by giotiskl**
Fixes the behavior of all ecommerce apps for missing products

### CODE REVIEW

File: apps/commercetools/src/Editor/CategoryPreviews/SortableListItem.tsx [hunk 1]
```diff
@@ -71,48 +76,47 @@ const styles = {
...
     return (
       <Card className={styles.card}>
         <>
           {isSortable && <CardDragHandle />}
           <section className={styles.description}>
             <Typography>
-              <Heading className={styles.name(category.name)}>
-                {categoryIsMissing ? category.id : category.name}
+              <Heading className={styles.heading(category)}>
+                {category.isMissing || !category.name
```
Comment (davidfateh) [inline apps/commercetools/src/Editor/CategoryPreviews/SortableListItem.tsx]: Optional: you use the same bool logic here as you do when you pass it into the `styles.heading` func. You could have that logic in var and reuse it.

File: apps/shopify/src/Pagination.js [hunk 2]
```diff
@@ -17,7 +14,7 @@ class Pagination {
 
   variants = [];
 
-  prevSearch = "";
+  prevSearch = '';
```
Comment (davidfateh) [inline apps/shopify/src/Pagination.js]: 🤣

File: apps/commercetools/src/AppConfig/index.spec.tsx [hunk 3]
```diff
@@ -70,8 +70,8 @@ describe("AppConfig", () => {
       [/commercetools Project Key/, ""],
       [/Client ID/, ""],
       [/Client Secret/, ""],
-      [/^API Endpoint/, ""],
```
Comment (jelz) [inline apps/commercetools/src/AppConfig/index.spec.tsx]: you've got `singleQuote: true` in prettierrc above, why wasn't this fixed?

File: apps/shopify/src/productResolvers.js [hunk 4]
```diff
@@ -30,11 +33,23 @@ export const fetchProductPreviews = async (skus, config) => {
...
+    .map(sku => {
+      try {
+        // If not valid base64 window.atob will throw
+        const unencodedId = atob(sku);
+        return { unencodedId, sku };
+      } catch (error) {
+        return null;
+      }
+    })
+    .filter(sku => sku && /^gid.*ProductVariant/.test(sku.unencodedId))
```
Comment (davidfateh) [inline apps/shopify/src/productResolvers.js]: What exactly is the regex testing? It seems like this functionality takes the skus, decodes them, tests the decoded value but then uses the encoded value finally.

Comment (giotiskl) [reply apps/shopify/src/productResolvers.js]: that's correct, the encoded value is what has to be passed in to the graphql query, but i need to decode them to determine which ones are validly formatted SKUs, then the rest (which are not) are put into the invalid SKUs array to be rendered as "missing products"

File: apps/commercetools/src/AppConfig/index.spec.tsx (hunk 3 above)
Comment (giotiskl) [reply apps/commercetools/src/AppConfig/index.spec.tsx]: I didn't run prettier on this file.. prettier is messed up all over the new apps repo

**Pull request merged**